shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -932,7 +932,7 @@ public int touch(TachyonURI path) throws IOException {
    */
   public int unpin(TachyonURI path) throws IOException {
     try {
-      mTfs.setPin(mTfs.open(path), false);
+      mTfs.setPin(mTfs.open(path), true);
```

Comment:
@calvinjia Because the java doc comment of this function says this function work recursively on a folder. We need to make a decision on this function's behavior.